src/EventSubscriber/GeneratedUserIngestControllerSubscriber.php

```diff
@@ -110,6 +117,10 @@ public function onLineItemUpdated(LineItemUpdated $event): void
             $groupResolver
         );
 
+        if (!$this->folderSyncEnabled) {
+            return;
+        }
```

Comment:
early return is good when you want to interrupt a function when something goes wrong or doesn't met some conditions no matter what. but for skipping a single line (even though it's the last one :) I think the classic way is better
```
        if ($this->folderSyncEnabled) {
            $this->userFolderSync->sync($date);
        }
```